libyul/AsmPrinter.cpp

```diff
@@ -261,7 +261,6 @@ string AsmPrinter::appendTypeName(YulString _type, bool _isBoolLiteral) const
 string AsmPrinter::formatSourceLocationComment(
```

Comment:
It no longer produces a comment so I think it should be renamed:
```suggestion
string AsmPrinter::formatSourceLocation(
```